docs/dev/api_concepts.md

```diff
@@ -896,3 +896,54 @@ result type ```R``` for the final result. E.g. for a histogram, ```V``` is a num
 
 {% top %}
 
+## EXPERIMENTAL: Reinterpreting a pre-partitioned data stream as keyed stream
```

Comment:
It might be better to add a dedicated "experimental" section in the DataStream API documentation for this. What do you think?

Reply:
+1